gamehub-frontend/src/pages/LandingPage/LandingPage.css

```diff
@@ -0,0 +1,156 @@
+html, body {
+    margin: 0;
+    padding: 0;
+    height: 100%;
+}
+
+.landing-container {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh; /* Ensure container covers the full viewport height */
```

Comment:
Looks good to me :)

Just in the line 10, 17 and 26 in this file, there are some comment. If you need it, keep it, otherwise, it's also good to remove it :)